client/controllers/events/eventFiltration.coffee

```diff
@@ -6,6 +6,42 @@ formatDateForInput = (date) ->
     return moment()
   if date.getTime then date else new moment(date)
 
+# Update input classes based on state of button: unselected, positive, negative
+updateInputState = (event) ->
```

Comment:
It seems like this should be handled by the jade template.

Reply:
Good point. I've been working on a refactor that reduces storing the state of the buttons in the DOM. It takes care of this issue but currently running into an issue with conflicting events. I'll push an update ASAP.